src/main/java/ch/njol/skript/expressions/ExprTimeDifference.java

```diff
@@ -31,14 +34,27 @@
 import ch.njol.skript.util.Date;
 import ch.njol.skript.util.Timespan;
 
-@Name("Time Since")
-@Description("The time that has passed since a date. If the given date is in the future, a value will not be returned.")
-@Examples("send \"%time since 5 minecraft days ago% has passed since 5 minecraft days ago!\" to player")
-@Since("2.5")
-public class ExprTimeSince extends SimplePropertyExpression<Date, Timespan> {
+@Name("Time Since/Until Dates")
+@Description("The time since a date has passed or the time until a date will pass.")
+@Examples({
+	"send \"%time since 5 minecraft days ago% has passed since 5 minecraft days ago!\" to player",
+	"send \"%time until {countdownEnd}% remaining of the countdown!\" to player"
+})
+@Since("2.5, INSERT VERSION (time until)")
+public class ExprTimeDifference extends SimplePropertyExpression<Date, Timespan> {
 
 	static {
-		Skript.registerExpression(ExprTimeSince.class, Timespan.class, ExpressionType.PROPERTY, "[the] time since %dates%");
+		Skript.registerExpression(ExprTimeDifference.class, Timespan.class, ExpressionType.PROPERTY,
+			"[the] time since %dates%",
+			"[the] [remaining] time [remaining] until %dates%");
+	}
+
+	private boolean sinceDate;
+
+	@Override
+	public boolean init(final Expression<?>[] exprs, final int matchedPattern, final Kleenean isDelayed, final SkriptParser.ParseResult parseResult) {
```

Comment:
Remove `final` modifiers from method params and import ParseResult directly
```suggestion
	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
```